dotcom-rendering/src/components/Picture.tsx

```diff
@@ -348,13 +360,14 @@ export const Picture = ({
 
 	const sources = generateSources(
 		master,
-		decideImageWidths({
-			role,
-			format,
-			isMainMedia,
-			isLightbox,
-			orientation,
-		}),
+		imageWidths ??
+			decideImageWidths({
+				role,
+				format,
+				isMainMedia,
+				isLightbox,
+				orientation,
+			}),
```

Comment:
So, if we're going to add more customisation to the `Picture` component, do we want to do it like this, where we add an override that says _anything goes_, or do we want to add a new flag for `isPodcastCover` or something like that?

Reason I ask is because of the [disclaimer at the top of `decideImageWidths`](https://github.com/guardian/dotcom-rendering/blob/391ded91bf1eee4d19d7a26d6316faafac0d3be5/dotcom-rendering/src/components/Picture.tsx#L48C4-L49C27):

```
All business logic for image sizing is contained in this one function. This is the source of truth.
```

With this change, this is no longer the case. I'll admit it doesn't feel sustainable to have a function that receives a growing series of params to account for weird and wonderful new use cases, it also doesn't feel good for that function to canonise some widths as "source of truth" and for everything else, there's a custom role. And since this podcast cover image [has been around for at least 8 years](https://github.com/guardian/content-api/pull/1294/files#diff-b49f77f39b2863c37d911e950a163ac0e5cf31ba0ec853d34f9b0f282d34f46aR29) and is canonised in CAPI and tag manager, maybe dotcom-rendering needs to bite the bullet and accept it as a thing also.

Reply:
The extra argument way you suggested seems to be how many of these components work in DCR. Also I don't want to break that contract about the source of truth. I will change it happily to use a prop through the Picture component as suggested. 
![image](https://github.com/user-attachments/assets/9c211cc5-a7fd-4256-aaae-d2de3d0c2404)